src/lib/Preview.js

```diff
@@ -1043,13 +1123,18 @@ class Preview extends EventEmitter {
             // Bump up preview count
             this.count.success += 1;
 
+            const metrics = this.fileMetrics.done(this.count);
+
             // Finally emit the viewer instance back with a load event
             this.emit('load', {
                 viewer: this.viewer,
-                metrics: this.logger.done(this.count),
+                metrics,
                 file: this.file
             });
 
+            // Track in interal logger
+            this.logger.metric(METRIC_FILE_PREVIEW_SUCCESS, metrics);
```

Comment:
I know I keep bringing this up, but why not just send the `metric` stuff from the logger to Gen204 instead of maintaining both `Logger`/`LogLevel` and `FileMetrics`? It looks like we repeat a lot of function calls and here we are directly passing in our `FileMetrics` info into `LogLevel`.